tests/runtests.py

```diff
@@ -629,16 +629,14 @@ def test_cmd_trace_run(self):
         return
 
     def test_cmd_unicorn_emulate(self):
-        before = ["br *main", "r", "si"]  # "si" needed to avoid invalid memory read
-
         nb_insn = 4
         cmd = "emu {}".format(nb_insn)
         res = gdb_run_cmd(cmd)
         self.assertFailIfInactiveSession(res)
 
         start_marker = "= Starting emulation ="
         end_marker = "Final registers"
-        res = gdb_run_cmd(cmd, before=before)
+        res = gdb_start_silent_cmd(cmd)
```

Comment:
this will not work as the "si" is needed to not run into a invalid memory read for `mov    rax, QWORD PTR fs:0x28` (the fs:0x28 is the problem here as it seems like)